src/mon/Monitor.cc

```diff
@@ -2832,6 +2832,14 @@ void Monitor::do_health_to_clog(bool force)
       level == health_status_cache.overall)
     return;
   clog->health(level) << "overall " << summary;
+  if (level != HEALTH_OK) {
+    Formatter *f = Formatter::create("json-pretty");
```

Comment:
if we already know the type of `Formatter`, why not just use `JSONFormatter`?

Reply:
Using JSONFormatter now.